README.md

```diff
@@ -42,6 +42,24 @@ $ cd chanjo-report
 $ pip install --editable .
 ```
 
+### Docker
+
+To run a local demo with Docker, ensure you have a local Docker running.
```

Comment:
I liked [my docs](#47 ) better for this part, because they were explaining a bit more what the commands did

Reply:
Less is more. 😉 I'll make a compromise. Or we can have a TL;DR.